plugins/alerts/frontend/public/javascripts/countly.views.js

```diff
@@ -12,811 +12,483 @@
-
-            // var emailList = [countlyGlobal.member._id];
-            var emailList = [];
-            $("#email-list-input  :selected").each(function() {
-                emailList.push($(this).val());
-            });
-            settings.alertValues = emailList && emailList.length > 0 ? emailList : null;
-            var currentId = $("#current_alert_id").text();
-            currentId && (settings._id = currentId);
-            return settings;
+        refresh: function () {
+           // this.$store.dispatch("countlyHooks/table/fetchAll");
         },
-        checkDisabled: function() {
-            var alertConfig = this.getWidgetSettings();
-            if (!alertConfig.selectedApps) {
-                $("#single-target-dropdown").addClass("disabled");
-                $("#single-target2-dropdown").addClass("disabled");
-                $("#single-target-condition-dropdown").addClass("disabled");
-                $("#alert-compare-value").addClass("disabled");
-                $("#alert-compare-value-input").attr("disabled", "true");
-            }
-            else {
-                $("#single-target-dropdown").removeClass("disabled");
-                $("#single-target2-dropdown").removeClass("disabled");
-                $("#single-target-condition-dropdown").removeClass("disabled");
-                $("#alert-compare-value").removeClass("disabled");
-                $("#alert-compare-value-input").removeAttr("disabled");
-            }
-
-            $("#create-widget").removeClass("disabled");
-            $("#save-widget").removeClass("disabled");
-            for (var key in alertConfig) {
-                if (!alertConfig[key]) {
-                    $("#create-widget").addClass("disabled");
-                    $("#save-widget").addClass("disabled");
-                }
-            }
-        }
     }
 });
 
-app.alertsView = new window.AlertsView();
+var AlertsHomeViewComponent = countlyVue.views.BaseView.extend({
+    template: "#alerts-home",
+    mixins: [countlyVue.mixins.hasDrawers("home")],
+    components: {
+        "table-view": TableView,
+        "drawer": AlertDrawer,
+    },
+    computed: {
+        countData: function () {
+            var count = this.$store.getters["countlyAlerts/table/count"];
+            return [
+               {label:'alert.RUNNING_ALERTS', value: count.r},
+               {label:'alert.TOTAL_ALERTS_SENT', value: count.t},
+               {label:'alert.ALERTS_SENT_TODAY', value: count.today},
+            ]
+        }
+    },
+    data: function () {
+        return {
+            canCreate: countlyAuth.validateCreate(app.alertsView.featureName)
+        }
+    },
+    beforeCreate: function () {
+       this.$store.dispatch("countlyAlerts/initialize");
+    },
+    methods: {
+        createAlert: function () {
+            this.openDrawer("home", countlyAlerts.defaultDrawerConfigValue());
+        },
+    },
+});
+app.alertsView = new countlyVue.views.BackboneWrapper({
```

Comment:
no need to add the returned backbone view to app.
infact place the entire js in this file within 
(function(){
//all code here.
})()

Reply:
follow this pattern please - 
https://github.com/Countly/countly-server/blob/13a6f4b304ded1b439fd9623cc208bc605f58dbb/plugins/slipping-away-users/frontend/public/javascripts/countly.views.js#L130-L148